src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component, ReactNode } from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  name: string,
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  clockTimerId = 0;
+
+  componentDidMount() {
+    this.clockTimerId = window.setInterval(() => {
+      const { today } = this.state;
+      const newToday = new Date(today.getTime() + 1000);
+
+      // eslint-disable-next-line no-console
+      console.info(newToday.toUTCString().slice(-12, -4));
```

Comment:
```suggestion
      // eslint-disable-next-line no-console
     window.console.info(newToday.toUTCString().slice(-12, -4));
```
Hint: if you will use window before the console you do not need to disable the linter